### PR TITLE
Small fixes in Meadow 2.0 for clarity and usability

### PR DESCRIPTION
**Pull request by palas**
See issues #798, #799, and #800.

### CODE REVIEW

File: meadow-client/src/Simulation.purs [hunk 1]
```diff
@@ -605,26 +615,36 @@ statePane state = div [ class_ $ ClassName "col"
...
+                          ]
+                    , row_ [ if (Map.size mState.oracles == 0)
+                             then text "No oracle values have been recorded"
+                             else renderOracles mState.oracles
+                           ]
+                    ]
                 ]
         ]
   where (State mState) = state.marloweState.state
+        (CommitInfo commits) = mState.commits
```
Comment (shmish111) [inline meadow-client/src/Simulation.purs]: where is this used?

File: meadow-client/src/Simulation.purs [hunk 2]
```diff
@@ -605,26 +615,36 @@ statePane state = div [ class_ $ ClassName "col"
...
-                                   ]
-                            , h3_ [ text "Oracle values"
-                                  ]
-                            , row_ [ renderOracles mState.oracles
-                                   ]
-                            ]
+      ] [ card_ [ cardBody_
+                    [ h3_ [ text "Money owed"
+                          ]
+                    , row_ [ if (Map.size commits.redeemedPerPerson == 0)
```
Comment (palas) [inline meadow-client/src/Simulation.purs]: Here

File: meadow-client/src/Simulation.purs [hunk 3]
```diff
@@ -605,26 +615,36 @@ statePane state = div [ class_ $ ClassName "col"
...
+                             then text "No participant is owed money"
+                             else renderMoneyOwed mState.commits
+                           ]
+                    , h3_ [ text "Commits"
+                          ]
+                    , row_ [ if (Map.size commits.currentCommitsById == 0)
```
Comment (palas) [inline meadow-client/src/Simulation.purs]: And here

File: meadow-client/src/Simulation.purs (hunk 1 above)
Comment (palas) [reply meadow-client/src/Simulation.purs]: in line 621

**Pull request merged**